native/membrane_video_compositor_common/src/plugins/transformation.rs

```diff
@@ -0,0 +1,72 @@
+use std::{any::Any, sync::Arc};
+
+use crate::WgpuContext;
+
+// Question to the reviewers: should the example below be in the finished documentation? Should it be
+//                            kept until the end of the plugins PR?
```

Comment:
I would definitely leave it here right now, once the rest of the system is complete we can either put it in the documentation, or at least use it somewhere in tests (but I would opt for the first one)

Reply:
Please leave this unresolved to allow other reviewers to comment on this

Reply:
IMO we can leave it here